test/elastic_record/log_subscriber_test.rb

```diff
@@ -16,7 +16,7 @@ def setup
   # end
 
   def test_request_notification
-    FakeWeb.register_uri(:any, %r[/test], status: ["200", "OK"], body: Oj.dump('the' => 'response'))
+    stub_request(:any, "#{Widget.elastic_connection.servers.first}/test").to_return(status: 200, body: Oj.dump('the' => 'response'))
```

Comment:
Fixed in https://github.com/data-axle/elastic_record/pull/46/commits/137508b3f002f8eb2fa378e9c3a264a227cc30b4 - I tried removing `stub_es_request` entirely but that passed through all of the requests, which is what we don't want.